doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -645,16 +652,66 @@ indexterm:[Action,Property,on-fail]
 
 |role
 |
-|This option only makes sense for recurring operations.  It restricts
- the operation to a specific role.  The truely paranoid can even
- specify +role=Stopped+ which allows the cluster to detect an admin
- that manually started cluster services.
- Allowed values: +Stopped+, +Started+, +Slave+, +Master+.
+|Run the operation only on node(s) that the cluster thinks should be in
+ the specified role. This only makes sense for recurring monitor operations.
+ Allowed (case-sensitive) values: +Stopped+, +Started+, and in the
+ case of <<s-resource-multistate,multi-state>> resources, +Slave+ and +Master+.
  indexterm:[role,Action Property]
  indexterm:[Action,Property,role]
 
 |=========================================================
 
+=== Monitoring Resources for Failure ===
+
+When Pacemaker first starts a resource, it runs one-time monitor operations
+(referred to as 'probes') to ensure the resource is running where it's
+supposed to be, and not running where it's not supposed to be. (This behavior
+can be affected by the +enable-startup-probes+ cluster option and the
++resource-discovery+ location constraint property.)
+
+Other than those initial probes, Pacemaker will not (by default) check that
+the resource continues to stay healthy. As in the example above, you must
+configure monitor operations explicitly to perform these checks.
+
+By default, a monitor operation will ensure that the resource is running
+where it is supposed to. The +target-role+ property can be used for further
+checking.
```

Comment:
When unpacking a resource in the PE, if there are no recurring monitor ops defined AND `$[some_condition}`, create a fake one.
We need `$[some_condition}` otherwise there would be no way to turn it off is for some reason you really didn't want any monitoring to occur.  I can't think what that might be though.

Unless... we could introduce a global option `default-monitor-interval` which defaults to `5min` and someone can set that to 0 if they really want it off.